ansible/roles/nim-waku/templates/docker-compose.yml.j2

```diff
@@ -18,11 +18,15 @@ services:
         condition: '{{ nim_waku_restart_condition }}'
     volumes:
       - '{{ nim_waku_node_db_path }}:/data'
+{% if nim_waku_websocket_enabled %}
+      - '{{ nim_waku_websocket_ssl_dir }}:{{ nim_waku_websocket_ssl_dir | mandatory }}'
```

Comment:
This is good, but these variables should also be present in the `defaults.yml` file with websocket disabled.
Similar to how websickify ones are defined:
https://github.com/status-im/infra-nim-waku/blob/d74fe01c8138e45604d0d897890fea68545db9ee/ansible/roles/nim-waku/defaults/main.yml#L50-L60

Reply:
added in 0392d62c41b34530314ddc20cade3a7d0f5365c3